### DISCUSSION TITLE
Idle timer/time out

### DISCUSSION THREAD

**Discussion by miffweaver** [SafeExamBrowser/seb-mac#305] [Q&A]
Hi! 

I've had a good look around before posting this and haven't been able to find an answer...

Is there a way of adding a timer to SEB to revert it back to a specific webpage after it reaches the threshold? I've spent a good amount of time looking through the settings (on macOS) but haven't found anything. I've used ecrisper in the past which is great for this, but support has ended and it's struggling to present certain webpages (scrolling is staggering).

If anyone has any other ideas as to how I might be able to achieve this I'd love to hear :)

Thanks

**Comment by danschlet** [comment]
Hi,

This isn't a use case we've been thinking of until now, because when doing exams/tests/quizzes, those would implement a timer for the maximum time granted to take a test.

So you could implement a timer on the website you're displaying and either redirect to the specific webpage or invoke the SEB Quit Link/URL which you would set in SEB's settings. With the Quit Link, you can either quit SEB completely, restart the session to the same Start URL or end an exam session and reload SEB's client settings (if those have a quit password set).

But all of this requires code in the website/webapplication you're displaying, there is no timer implemented in SEB itself.
